src/installer/installer.go

```diff
@@ -439,6 +443,23 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	return nil
 }
 
+func (i *installer) waitForETCDBootstrap(ctx context.Context) error {
+	i.UpdateHostInstallProgress(models.HostStageWaitingForBootkube, "waiting for ETCD bootstrap to be complete")
+	i.log.Infof("Started waiting for ETCD bootstrap to complete")
+	return utils.WaitForPredicate(5*time.Minute, generalWaitInterval, func() bool {
```

Comment:
Let's change this to:
```suggestion
	return utils.WaitForPredicate(waitForeverTimeout, generalWaitInterval, func() bool {
```
I know it shouldn't take that long but if there's a chance that etcd will complete the bootstrap after 10 minutes we don't want this timeout to be the reason the installation failed.
In general the installer isn't setting timeouts for the installation the assusted-service does.

Reply:
Makes sense! Accepted suggested changes. Thank you!